src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasures.test.ts

```diff
@@ -0,0 +1,92 @@
+import { migrateCalculatedMeasures } from "./migrateCalculatedMeasures";
+import { dataModelsForTests } from "@activeviam/data-model-5.0";
+import { contentServer } from "../__test_resources__/contentServer";
+import { uiCalculatedMeasuresFolder } from "../__test_resources__/uiCalculatedMeasuresFolder";
+import { uiDashboardsFolder } from "../__test_resources__/uiDashboardsFolder";
+import { uiWidgetsFolder } from "../__test_resources__/uiWidgetsFolder";
+import _cloneDeep from "lodash/cloneDeep";
+
+const dataModel = dataModelsForTests.sandbox;
+
+const contentServerForTests = _cloneDeep(contentServer);
+
+contentServerForTests.children!.ui.children = {
+  ...contentServerForTests.children!.ui.children,
+  calculated_measures: uiCalculatedMeasuresFolder,
+  dashboards: uiDashboardsFolder,
+  widgets: uiWidgetsFolder,
+};
```

Comment:
I did this because the `widgets` in the `contentServer` test resource contain no calculated measures, and the `dashboards` contain only one which is not in the `calculated_measures` folder. None of the `calculated_measures` from the `contentServer` test resource would have been migrated because they aren't used in any of the saved widgets or dashboards. So in order to test this function properly, I need the `uiWidgetsFolder`, `uiDashboardsFolder` and `uiCalculatedMeasuresFolder`. But maybe `_cloneDeep` is not actually necessary ?

Reply:
Makes sense.
Ideally, it would be nice if we could harmonize the test resources and keep only `contentServer`: keep only a minimal amount in there, but make sure that it covers the calculated measures as well.
It's not the priority though, so we can harmonize the test resources in a separate PR.
In the meantime, what you did is good.